hub/auto/unstructured/kaggle.py

```diff
@@ -32,7 +32,7 @@ def _set_environment_credentials_if_none(kaggle_credentials: dict = None):
             os.environ[ENV_KAGGLE_KEY] = key
 
 
-def download_kaggle_dataset(tag: str, local_path: str, kaggle_credentials: dict = {}):
+def download_kaggle_dataset(tag: str, local_path: str, kaggle_credentials: dict = None):
```

Comment:
I just noticed, _set_environment_credential_if_none will ignore a dictionary of kaggle credentials passed as it only goes into the if condition if the credentials are None. I think the if statement on L23 should probably be outside the if on L21.